From 1f3bc6833d192dff5969c9901557ad8e22b3a689 Mon Sep 17 00:00:00 2001
From: Christoph Siegert <christoph.siegert@uni-leipzig.de>
Date: Fri, 21 Jan 2022 13:58:17 +0100
Subject: [PATCH] do not recompile on "make test" and "make install"

This patch removes ".PHONY" for version.h.
".PHONY version.h" causes make to not look for the version.h file so
everything that depends on it will be rebuilt on every run of make.
This causes
    make LIBBIGWIG=/some/path/libBigWig.a
    make test
to fail as "make test" will try to recompile MethylDackel without
LIBBIGWIG set to /some/path/libBigWig.a
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1352d5f..911da1f 100644
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ LIBS ?=  # e.g., -L$PREFIX/lib, or where ever htslib is
 LIBBIGWIG ?=
 CFLAGS ?= -Wall -g -O3 -pthread
 
-.PHONY: all clean install version.h
+.PHONY: all clean install
 
 .SUFFIXES:.c .o
 
